data/pathfinder/paizo/roleplaying_game/bestiary/b1_abilities_race.lst

```diff
@@ -1104,9 +1113,9 @@ Jet			KEY:Squid ~ Jet		CATEGORY:Special Ability	TYPE:SpecialQuality.Extraordinar
 
 ###Block: Stirge Abilities - in bestiary_core
 # Ability Name	Unique Key				Category of Ability		Type						Description																																																																																											Source Page
-Attach		KEY:Stirge ~ Attach		CATEGORY:Special Ability	TYPE:SpecialAttack.Extraordinary	DESC:When a stirge hits with a touch attack, its barbed legs latch onto the target, anchoring it in place. An attached stirge is effectively grappling its prey. The stirge loses its Dexterity bonus to AC, but holds on with great tenacity and inserts its proboscis into the grappled target's flesh. A stirge has a +8 racial bonus to maintain its grapple on a foe once it is attached. An attached stirge can be struck with a weapon or grappled itself. If its prey manages to win a grapple check or Escape Artist check against it, the stirge is removed.	SOURCEPAGE:p.260
-Blood Drain		KEY:Stirge ~ Blood Drain	CATEGORY:Special Ability	TYPE:SpecialAttack.Extraordinary	DESC:A stirge drains blood at the end of its turn if it is attached to a foe, inflicting 1 point of Constitution damage. Once a stirge has dealt 4 points of Constitution damage, it detaches and flies off to digest the meal. If its victim dies before the stirge's appetite has been sated, the stirge detaches and seeks a new target.																																					SOURCEPAGE:p.260
-Diseased		KEY:Stirge ~ Diseased		CATEGORY:Special Ability	TYPE:SpecialQuality.Extraordinary	DESC:Any creature subjected to a stirge's blood drain attack has a 10%% chance of being exposed to filth fever, blinding sickness, or a similar disease. Once this check is made, the victim can no longer be infected by this particular stirge, though attacks by different stirges are resolved normally and may result in multiple illnesses.																																				SOURCEPAGE:p.260
+#Attach		KEY:Stirge ~ Attach		CATEGORY:Special Ability	TYPE:SpecialAttack.Extraordinary	DESC:When a stirge hits with a touch attack, its barbed legs latch onto the target, anchoring it in place. An attached stirge is effectively grappling its prey. The stirge loses its Dexterity bonus to AC, but holds on with great tenacity and inserts its proboscis into the grappled target's flesh. A stirge has a +8 racial bonus to maintain its grapple on a foe once it is attached. An attached stirge can be struck with a weapon or grappled itself. If its prey manages to win a grapple check or Escape Artist check against it, the stirge is removed.	SOURCEPAGE:p.260
```

Comment:
Why are these being commented out?

Reply:
It's moved to ce_abilities_familiar_race_cr.lst.